docs/developersguide/Codestyle.md

```diff
@@ -0,0 +1,211 @@
+# Codestyle
+
+We use the following code conventions for [Java](#Java) and [JavaScript](#JavaScript).
+
+## Java
+The OpenTripPlanner Java code style is revised in OTP2. We use the Google Java style guide, with a 
+few modifications. Here is the original Google style guide: 
+https://google.github.io/styleguide/javaguide.html
+
+### IntellJ Code Style formatter 
+If you use IntelliJ, import the provided [intellij-code-style.xml](../../intellij-code-style.xml). 
```

Comment:
This PR contains both `intellij-code-style.xml` and `intellij-java-style.xml`, is this intentional? What is the difference?

Reply:
The intellij-java-style.xml should not be there, it is the Google original, which I tuned and saved as _intellij-code-style.xml_ (It apply to more than Java code).